alluxio/alluxio.sh

```diff
@@ -103,10 +102,19 @@ function bootstrap_alluxio() {
   sudo ln -s "${ALLUXIO_HOME}/client/alluxio-client.jar" ${HIVE_HOME}/lib/alluxio-client.jar
   sudo mkdir -p ${HADOOP_HOME}/lib/
   sudo ln -s "${ALLUXIO_HOME}/client/alluxio-client.jar" ${HADOOP_HOME}/lib/alluxio-client.jar
-  sudo mkdir -p ${PRESTO_HOME}/plugin/hive-hadoop2/
-  sudo ln -s "${ALLUXIO_HOME}/client/alluxio-client.jar" ${PRESTO_HOME}/plugin/hive-hadoop2/alluxio-client.jar
   sudo systemctl restart hive-metastore
   sudo systemctl restart hive-server2
+
+  # Optionally configure presto
+  # OK to fail in this section
+  set +o errexit
+  PRESTO_HOME=${PRESTO_HOME:-$(ls -d -- /presto-server*)}
+  if [ ! -z $PRESTO_HOME ]; then
+    sudo mkdir -p ${PRESTO_HOME}/plugin/hive-hadoop2/
+    sudo ln -s "${ALLUXIO_HOME}/client/alluxio-client.jar" ${PRESTO_HOME}/plugin/hive-hadoop2/alluxio-client.jar
+    sudo systemctl restart presto
```

Comment:
Can it be Presto hasn't been configured yet at this point?

Reply:
Correct, it is possible. In that case `PRESTO_HOME` should not be defined and we will not run this line. However if presto is to be installed, it should be installed before Alluxio.

My assumption is that init actions are executed in sequence and the caveat here is that the presto init action should be specified before the Alluxio init action.

I didn't like this requirement but seems like some flavors of Presto installs at `presto-server-<version>` and others at `presto-server`. To handle the former, the order of execution becomes important. Let me know if you think there's a better way.

Reply:
> caveat here is that the presto init action should be specified before the Alluxio init action.

that's counterintuitive, considering the fact that the end goal is that Presto uses Alluxio and not vice versa.

Reply:
Yes, init actions are executed sequentially. We need to document this dependency on Presto in the README, if it can not be avoided.

Reply:
Added a line to the README